book/website/collaboration/new-community/new-recruitment.md

```diff
@@ -0,0 +1,524 @@
+    * Consider unconscious bias against all characteristics - often unconscious bias moves beyond the nine protected characteristics and into dimensions such as education level, socioeconomic status, geography, and hobbies.
+        * Avoid superlatives
+        * Check for accessibility of advert using a screen reader
+        * Keep mandatory skills to a minimum and thoroughly consider whether all criteria are essential
+        * Avoid asking for mandatory travel, weekend work and overtime, if possible
+    * Remove all biased language in the job advert
+        * Consider using a gender decoder to identify words/phrases that may imply a specific gender and re-phrase if needed
+    * Provide clear guidelines & deadlines plus instructions & contacts for people requiring reasonable adjustments 
+        * Provide transparency over interview structure and application assessment criteria where possible 
+    * Include relevant policies clearly in the job advert 
+    * Be transparent about the compensation package
+        * Provide a starting salary or salary range 
+        * Include any statutory or enhanced benefits your organisation may offer such as paternity, maternity and parental leave and/or benefits such as time off for dependents, and annual leave entitlement. 
+    * Directly reference your organisational EDI commitment
+        * Include your organisational Diversity Statement where available.
+        * Include your organisational Values (if applicable). 
+- [ ] Post job advert 
+    * Consider the best places to advertise to reach a diverse pool of candidates
+        * See “ Resources” section for suggestions of advertising venues
+        * If certain demographics are often missing from your pool, utilise spaces that target those groups
+        * Consider reviewing the diversity of the pool attained from previous, similar advertising and seek specific advertising to reach other groups that were underrepresented 
+    * Use a gender decoder to review the advert after you have created it
+    * Leverage your networks, as well as those of others in the organisation, to share the opportunity advertised e.g. Slack, LinkedIn, word of mouth at conferences etc. 
+        * When utilising your networks, consider if there are any groups potentially missing. Consider using alternative spaces (examples listed in Resources section) to reach groups that aren’t in your network. 
+- [ ] Set a suitable closing deadline
+    * Ensure a closing date is included on the advert
+        * Advertise roles minimally for 2-4 weeks where possible
+    * Where possible, stick to the advertised close date. Avoid stating that the role will be closed early if a suitable candidate is identified.  
+    * Consider the day of the week and time of closing your advert. For example, a closing date set on a weekend may be valuable for people who have limited free time.
+    * Be explicit on the time zone which will be used to close the application period 
+    * 
```

Comment:
Advertise the advert: Some HR departments insist on unwieldy opening paragraphs, or otherwise not very inviting and inclusive structures of the adverts. It can be very helpful when advertising it on social media and on other websites to have a short (1-2 sentences) summary of the post, a (short) list of essential criteria, salary and closing date, and maybe a sentence about the team they will work with, and a contact. This can encourage people to open the job ad and look beyond the HR template.